### PR TITLE
for wally trends figure

### PR DESCRIPTION
**Pull request by jordansread**
simplifying this fig and scripting mobile #75

### CODE REVIEW

File: data/siteText.yaml [hunk 1]
```diff
@@ -142,7 +142,7 @@ footer-text: >-
   United States Geological Survey National Climate Change and Wildlife Science Center grant 10909172 to the University of Wisconsin–Madison.
 footer-team: >-  
   This project is led by Jordan S Read of the U.S. Geological Survey. The project team includes Luke Winslow (USGS), Gretchen Hansen (WI-DNR; MN-DNR), 
-   Pete Jacobson (MN-DNR), Emily Stanley (University of Wisconsin - Madison), Matt Diebel (WI-DNR), and Kevin Wehrly (MI-DNR). 
+   Pete Jacobson (MN-DNR), Emily Stanley (University of Wisconsin - Madison), Matt Diebel (WI-DNR), Megan Hines (USGS), and Kevin Wehrly (MI-DNR).
```
Comment (mhines-usgs) [inline data/siteText.yaml:145]: :D

**Comment by jordansread** [comment]
@mhines-usgs can you review while you're in there? :)

**Comment by mhines-usgs** [comment]
@jread-usgs sure, yeppers. checking things out now.

**Comment by mhines-usgs** [comment]
Looks ok, but I am not building it into a final product. Just reviewing this code. Did you forget to put the new svg into the repo that you reference in walleyeDecline.mustache? I don't see them in the code after I pulled your branch.

**Comment by jordansread** [comment]
yeah, we keep the figs out of the repo (except `sandbox`), so it just is the build scripts to make each fig.

**Comment by mhines-usgs** [comment]
Ok, I get it, seems ok to me, then!

**Pull request merged**